stable/insights-chart/values.yaml

```diff
@@ -54,3 +77,10 @@ insights:
         cpu: "25m"
       limits:
         memory: "512Mi"
+  metrics:
+    resources:
+      requests:
+        memory: "32Mi"
+        cpu: "25m"
+      limits:
+        memory: "512Mi"
```

Comment:
The limit seems high compared to the request, maybe it's ok, but just checking.

Reply:
Yes , I have to get a sweet spot